idmapping-rest/src/test/java/org/uniprot/api/idmapping/queue/UniParcIdMappingDownloadResultWriterTest.java

```diff
@@ -185,10 +179,6 @@ void canWriteResultFile() throws IOException {
         JsonNode warningNode = warnings.get(0);
         assertEquals("1", warningNode.findValue("code").asText());
         assertEquals("msg", warningNode.findValue("message").asText());
-
```

Comment:
I think we could also check if heart beat attributes were updated, right?
This comment also apply for uniprot and uniref tests